css/style.css

```diff
@@ -1,3 +1,103 @@
-h1 {
-  color: salmon;
+body {
+  background-color: #e9ebe7;
+  font-family: 'Hind', sans-serif;
+  color: #43474b;
+}
+
+.parent {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+}
+
+    .guideBooks, .weekend {
+      display: flex;
+      flex-direction: row;
+      font-size: 1.5em;
+      color: white;
+      text-align: center;
+      padding-bottom: 30px;
+    }
+
+            .sanFrancisco {
```

Comment:
When several elements share a lot of css properties, this becomes a good opportunity to keep your code DRY by pulling the repeated properties out into a single class:

```css
.city {
   background-size: cover;
   width: 220px;
   height: 250px;
   margin: 0 10px;
}

.sanFrancisco {
   background: url(../img/san-francisco.jpg);
}

.newYork {
   background: url(../img/new-york.jpg);
}
...
```